CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Cromwell Change Log
 
+## 50 Release Notes
+
+### New functionality
+
+#### xxHash for faster call-caching using file hashes
```

Comment:
Rather than putting all the detail in the changelog, could you document this in user facing docs (eg readthedocs) and then put a brief placeholder linking to that from here?

Otherwise, in a couple of versions time, it'll be really hard to direct anyone to the documentation for this feature.